lib/changed_rpm_files_helper.rb

```diff
@@ -52,7 +52,7 @@ def parse_rpm_changes_line(line)
       changes << "user" if expected_tag?("U", 5)
       changes << "group" if expected_tag?("G", 6)
       changes << "time" if expected_tag?("T", 7)
-      changes << "capabilities" if expected_tag?("P", 8)
+      changes << "capabilities" if @rpm_changes.size > 8 &&  expected_tag?(["P", "C"], 8)
```

Comment:
According to the rpm man page 'P' means 'capabilities differ'. Doesn't 'C' mean "SELinux context differs'?